docs-api/frontity-packages/features-packages/wp-comments.md

```diff
@@ -0,0 +1,261 @@
+---
+description: API reference of `@frontity/wp-comments` package
+---
+
+# `@frontity/wp-comments`
+
+Comments package that adds integration for WordPress native comments.
+
+<!-- toc -->
+
+- [Installation](#installation)
+- [Settings](#settings)
+  - [In WordPress](#in-wordpress)
+  - [In Frontity](#in-frontity)
+- [Usage](#usage)
+  - [Getting comments of a post](#getting-comments-of-a-post)
+  - [Sending new comments for a post](#sending-new-comments-for-a-post)
+- [API Reference](#api-reference)
+  - [Handlers](#handlers)
+    - [`@comments/:id`](#commentsid)
+  - [State](#state)
+    - [`state.comments.forms[]`](#state-comments-forms)
+    - [`state.source.comment[id]`](#state-source-comment-id)
+  - [Actions](#actions)
+    - [`actions.comments.updateFields()`](#actions-comments-updatefields)
+    - [`actions.comments.submit()`](#actions-comments-submit)
+- [Demo](#demo)
+
+<!-- tocstop -->
+
+## Installation
+
+Add the `wp-comments` package to your project:
+
+```bash
+npm i @frontity/wp-comments
+```
+
+## Settings
+
+### In WordPress
+
+In order to use this package, you will need to add a single line of configuration to your Wordpress installation:
+
+```php
+add_filter( 'rest_allow_anonymous_comments', '__return_true' );
+```
+
+[This filter](https://developer.wordpress.org/reference/hooks/rest_allow_anonymous_comments/) enables creating comments for anonymous users via the REST API.
+
+You can add this that directly in your theme's `functions.php` file or use a [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin.
+
+### In Frontity
+
+This package doesn't have any configuration. It just needs to be added to the `packages` array in `frontity.settings.js`.
+
+**`frontity.settings.js`**
+
+```js
+export default {
+  packages: ["@frontity/wp-comments"],
+};
+```
+
+## Usage
+
+### Getting comments of a post
+
+We can use the `@comments/:id` handler to fetch all the comments of a specific post (`actions.source.fetch("@comments/60")`). This data will be populated to the state so then we can do `state.source.get("@comments/60")` to get the ID's of these comments. With each ID we can get the details from the state at `state.source.comment[id]`
```

Comment:
Could we use a screen shot showing what the contents of the state look like - similar to what I did in [this page in the tutorial](https://tutorial.frontity.org/part3-displaying-posts/understanding-the-frontity-state)? 

It might make it easier to understand the description of the process if it can be related to a visualisation of the structure of the state.